<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(11, 0, 57);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(83, 100, 176);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_user_type_workflow">
        <property name="styleSheet">
         <string notr="true">background-color:rgb(4, 255, 0);color:rgb(117, 2, 2);font-size:12pt;</string>
        </property>
        <property name="text">
         <string>Operator</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>true</bold>
          <kerning>false</kerning>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(0, 157, 255);color:rgb(212, 255, 70);font-size:10pt;</string>
        </property>
        <property name="text">
         <string>New Order as Operator</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_of_entrances" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(83, 100, 176);</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="customer_new_order_operator">
        <property name="styleSheet">
         <string notr="true">color: rgb(149, 38, 30); background-color: rgb(241, 224, 255);font-size: 10pt;</string>
        </property>
        <property name="text">
         <string>Customer</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="customer_entrance_new_order_operator">
        <property name="styleSheet">
         <string notr="true">color: rgb(149, 38, 30); background-color:rgb(224, 255, 188); font-size:13pt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="marker1_new_order_operator">
        <property name="styleSheet">
         <string notr="true">color: rgb(149, 38, 30); background-color: rgb(241, 224, 255);font-size: 10pt;</string>
        </property>
        <property name="text">
         <string>Marker 1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="marker1_entrance_new_order_operator">
        <property name="styleSheet">
         <string notr="true">color: rgb(149, 38, 30); background-color:rgb(224, 255, 188); font-size:13pt;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="marker2_new_order_operator">
        <property name="styleSheet">
         <string notr="true">color: rgb(149, 38, 30); background-color: rgb(241, 224, 255);font-size: 10pt;</string>
        </property>
        <property name="text">
         <string>Marker 2</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="marker2_entrance_new_order_operator">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(149, 38, 30); background-color:rgb(224, 255, 188); font-size:13pt;</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="due_date_new_order_operator">
        <property name="styleSheet">
         <string notr="true">color: rgb(149, 38, 30); background-color: rgb(241, 224, 255);font-size: 10pt;</string>
        </property>
        <property name="text">
         <string>Due Date</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="due_date_entrance_new_order_operator">
        <property name="styleSheet">
         <string notr="true">color: rgb(149, 38, 30); background-color:rgb(224, 255, 188); font-size:13pt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="make_new_order_operator">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 117, 119); color: rgb(126, 255, 141); font-size:13pt;</string>
     </property>
     <property name="text">
      <string>MAKE NEW ORDER</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="feedback_label">
     <property name="styleSheet">
      <string notr="true">background-color:rgb(11, 0, 57);color:rgb(255, 0, 0);font-size:12pt;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="communications_entryfield_new_order_operator">
     <property name="styleSheet">
      <string notr="true">color: rgb(162, 0, 0); background-color: rgb(234, 234, 234); font: 700 9pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:700; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:400;&quot;&gt;Achtung: Alle Felder hier sind Pflichtfelder und müssen vom Operator ausgefüllt werden. Es erfolgt keine Meldung an den Vertrieb in Version 1.0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
